sycl/include/sycl/types.hpp

```diff
@@ -393,11 +439,7 @@ template <typename Type, int NumElements> class vec {
   }
   template <typename DataT_, typename T>
   static constexpr auto FlattenVecArgHelper(const T &A) {
-#if defined(__INTEL_PREVIEW_BREAKING_CHANGES)
     return std::array<DataT_, 1>{vec_data<DataT_>::get(static_cast<DataT_>(A))};
-#else
-    return std::array<DataT_, 1>{vec_data<DataT_>::get(A)};
-#endif
```

Comment:
Is there a reason we can promote this out of preview now? Was it just a mistake to have it under preview before?

Reply:
Correct. It was a mistake and shouldn't have been there to begin with.